src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,133 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import './TodoList.scss';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+export class TodoList extends React.Component {
+  state = {
+    selected: [],
+  }
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+  componentDidUpdate(prevProps) {
+    if (
+      prevProps.todos.length
+        !== this.props.todos.length
+        && prevProps.todos.length === 0
+    ) {
+      /* eslint-disable */ 
```

Comment:
why disable linter here?)

Reply:
You also need to check if prevState.selected !== this.props.todos
.filter(todo => todo.completed)
.map(todo => todo.id).